main.cpp

```diff
@@ -2,7 +2,9 @@
 
 #include "create_tridiagonal.hpp"
 #include "matrix_operations.hpp"
+#include "test_matrix_operations.hpp"
 
 int main(int argc, char *argv[]) {
+    test_max_offdiag_symmetric();
     return 0;
 }
```

Comment:
```suggestion
}

```

La til newline på slutten av filen